src/accounts/AccountPage.tsx

```diff
@@ -60,7 +71,8 @@ const AccountAboutPage: FC = () => {
   const onRenameAccountBtnClick = () => {
     handleRenameActiveAccount(activeAccount.id, activeAcctName)
   }
-  const shouldShowDeleteFreeAccountButton = CLOUD && account.type === 'free'
+
+  const shouldShowDeleteFreeAccountButton = CLOUD && accountDetails?.isDeletable
```

Comment:
For clarity, and to make sure we don't get weird behavior if the `isDeletable` property values change, I think this should be

`Boolean(CLOUD && accountDetails?.isDeletable)`

Reply:
yup, I think that will serve as a nice safeguard